src/textacy/constants.py

```diff
@@ -21,6 +21,51 @@
 OBJ_DEPS: set[str] = {"attr", "dobj", "dative", "oprd"}
 AUX_DEPS: set[str] = {"aux", "auxpass", "neg"}
 
+MIN_QUOTE_LENGTH: int=4
```

Comment:
i'm not sure this is a "constant" value, seems more like a parameter with a default value that should go in the `direct_quotations` extraction function. what do you think?

Reply:
yeah that makes sense